services/brig/src/Brig/API/Public.hs

```diff
@@ -818,6 +818,17 @@ sitemap o = do
     Doc.returns (Doc.ref Public.modelClient)
     Doc.response 200 "Client" Doc.end
 
+  get "/clients/:client/supported-features" (continue getClientSupportedFeaturesH) $
```

Comment:
Is this endpoint passed through in the [nginz config](https://github.com/zinfra/backend-wiki/wiki/Checklists#adding-new-endpoints-in-wire-server) ?

Reply:
Good point, thanks!  I'll add that.

Reply:
actually, it looks like we're cool:

```yaml
    - path: /clients
      envs:
      - all
```

Reply:
Perfect!